src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -743,20 +723,56 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string src
         }
     }
 
+    /// <summary>
+    /// Estimator for <see cref="KeyToVectorMappingTransformer"/>. Converts the key types back to their original vectors.
+    /// </summary>
     public sealed class KeyToVectorMappingEstimator : TrivialEstimator<KeyToVectorMappingTransformer>
     {
-        internal static class Defaults
+        public static class Defaults
```

Comment:
>public [](start = 8, length = 6)

why it became public?

Reply:
My bad! Thanks for pointing out.

---
In reply to: [253663731](https://github.com/dotnet/machinelearning/pull/2364#discussion_r253663731) [](ancestors = 253663731)